7/Constructor1/js/javascript.js

```diff
@@ -0,0 +1,12 @@
+function getData() {
+            var htmlPage;
+
+            var titleValue = title.value;
+            var descriptionValue = description.value;
+
+            htmlPage = "<h1>" + titleValue + "</h1>";
+            htmlPage += "<p>" + descriptionValue + "</p>";
+
+            document.body.innerHTML = htmlPage;
+}
```

Comment:
you can remove this file

Reply:
Yeah, I know that I can delete it, but I don't want to do it. It's mostly for memo, for experience. Just let it be :-)